libs/typings/src/ui/ui-events.ts

```diff
@@ -56,7 +56,6 @@ export const enum NotificationEvents {
  **/
 export const enum NotificationTypes {
   Info = 'info',
-  Alert = 'alert',
```

Comment:
why `Alert` type had to be removed?

Reply:
It was replaced with Info, per the updated design
https://www.figma.com/design/WbIHLctkDhGJjLKLhZcjCf/AKASHA-WORLD-DESIGN-SYSTEM?node-id=13-2121&t=CDxoD4c4899D6Wrj-0

Reply:
We now have only 4 notif types: Info, Caution, Success, Error